debezium.yaml

```diff
@@ -0,0 +1,136 @@
+package:
+  name: debezium
+  version: 3.0.1
+  epoch: 0
+  description: Debezium is a change data capture (CDC) platform that achieves its durability, reliability, and fault tolerance qualities by reusing Kafka and Kafka Connect.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-21-default-jvm
```

Comment:
Just curious if we want to require this version of java. My recollection was that we wanted to not necessarily specify a particular version for the package dependency. Maybe @pnasrat knows better, or can point to a better direction to ask.

Reply:
@vaikas  this very much depends on the package but yes @Dentrax  can drop the runtime dependency on the JDK to make this work cross JDKs.

I know @EyeCantCU has worked on various packages and images that support multijdk eg the ant image.

Reply:
That makes sense. Since Debezium [supports JDK>17](https://github.com/debezium/debezium/blob/95c552cdfd94de9f20ddb2c0ab8215bf42efd1ee/README.md?plain=1#L57), and b[ased on kafka-3](https://github.com/debezium/container-images/blob/d47b7f3a837d0cf0f911322e831159096b5dc629/connect-base/3.0/Dockerfile#L2C45-L2C50) image, which we [add openjdk-17](https://github.com/wolfi-dev/os/blob/e368fc4d7f98e91c3ef4e9ec4811c137da967008/kafka-3.8.yaml#L14) already in its runtime deps, we don't need to provide any openjdk deps in the runtime here.

Are we allowed to use openjdk-17 runtime with the package that build with openjdk-21?

Reply:
It depends on the package. I.E., Open Liberty builds with 21 and optionally 17, but can run on 8+. Unsure in Debezium's case